onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -484,6 +484,28 @@ def aten_arange_start(
     return result
 
 
+@torch_op("aten::arange.start_step", private=True)
+def _aten_arange_start_step_onnx(
```

Comment:
```suggestion
def _adjust_endpoints_for_arange_int_dtype(
```

or something more descriptive?